can/interfaces/socketcan/socketcan.py

```diff
@@ -316,6 +316,7 @@ class CyclicSendTask(
     def __init__(
         self,
         bcm_socket: socket.socket,
+        task_id: int,
```

Comment:
Please make sure a comment is added below for this parameter.

Reply:
Done.